### PR TITLE
Used array methods of array of object

### PR DESCRIPTION
**Pull request by Dove-white**
#Summary
## An array of objects, using some of the most commonly used array methods in JavaScript.

- forEach method to print each character's name.
- filter method to return a new array containing only the characters who are over 25 years old.
- map method to return a new array containing only the names of the characters.
- reduce method to calculate the total age of all the characters.
- sort method to return the characters by age from oldest to youngest.

### CODE REVIEW

**Comment by Dove-white** [comment]
https://docs.google.com/document/d/1maQanuHB7oCclI54iQCHjRUiLlC3_SRBMTxWvkTOkrU/edit?usp=sharing
This is the link to the question.

File: array_methods [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    { name: "Mr.A", age: 34, occcupation: "Gangster", weapon: "gun" },
+    { name: "Mr.B", age: 26, occcupation: "Pastor", weapon: "bible" },
+    { name: "Mr.C", age: 52, occcupation: "Banker", weapon: "money" },
+    { name: "Mr.D", age: 20, occcupation: "Boxer", weapon: "strenth" },
+  ];
+  
+  // forEach method
+  const forEachMethod = characters.forEach((propaty) => {
+    console.log(propaty.name);
+  });
```
Comment (yonas44) [inline array_methods:28]: Hey @Dove-white nice job using the `forEach` method, just as a tip, you can skip assigning the loop to the variable forEachMethod as it has no use in your code. You can simply loop through the characters and console the names.

Example:
```
characters.forEach((character) => {
    console.log(character.name);
  });
```

File: array_methods [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  
+  // filter method
+  const filterMethod = characters.filter((value) => value.age > 25);
+  
+  console.log('characters who are over 25 years old; ');
+  console.log(filterMethod);
```
Comment (yonas44) [inline array_methods:34]: Correct implementation of filter and nice use of variables as you can refer the new array using the variable name :+1:

**Review by yonas44 — APPROVED** [review]
Hey @Dove-white nice job implementing the methods, Keep up the good work! :heavy_check_mark:

File: array_methods (hunk 2 above)
Comment (Dove-white) [reply array_methods:34]: Thank you for the corrections!

**Pull request merged**